src/main/scala/uk/gov/nationalarchives/fileformat/FFIDExtractor.scala

```diff
@@ -30,8 +30,9 @@ class FFIDExtractor(sqsUtils: SQSUtils, config: Config) {
       val containerSignatureVersion = signatureOutput(1).split(" ").last
       val droidSignatureVersion = signatureOutput(2).split(" ").last
       val consignmentPath = s"""$efsRootLocation/${file.consignmentId}"""
+      val filePath = s""""$consignmentPath/${file.originalPath.replaceAll("\"", "\\\\\"")}""""
```

Comment:
Does the `File` API work here instead of trying to escape the name in bash? I see you're using it in https://github.com/nationalarchives/tdr-download-files/pull/9.

Reply:
I can't see a good way to use it. The download files lambda was calling `mkdir` and there's a direct equivalent in the file api but this command is a custom one to run droid. I did have a look at getting the `File` object to escape the characters but I couldn't get it to work.